python/xorbits/pandas/tests/test_accessors.py

```diff
@@ -140,3 +140,7 @@ def test_obj_docstring(setup, dummy_str_series, dummy_dt_series, dummy_df):
     assert dummy_df.iat.__doc__ is not None and dummy_df.iat.__doc__.endswith(
         "This docstring was copied from pandas."
     )
+
+
+def test_dir(setup):
```

Comment:
This test suite is mainly for accessors like `pd.Series.str` or `pd.Series.dt`. 

How about move this test case to `python/xorbits/pandas/pandas_adapters/tests/test_pandas_adapters.py`?